README.md

```diff
@@ -8,8 +8,7 @@
 [![SonarCloud Maintainability][sonarcloud-badge-maintainability]][sonarcloud-measure-maintainability]
 [![BCH compliance][bettercodehub-badge]][bettercodehub-results]
 
-Short for _jOOQ Loader_. A utility library to add basic object-relation mapping
-to your [jOOQ][jooq] code.
+Short for _jOOQ Loader_. A utility library to add basic object-relation mapping to your [jOOQ][jooq] code.
```

Comment:
Any reason we are switching to unwrapped text? MarkDown is a lot easier to read in plaintext viewers if it's wrapped.

Reply:
The text is still wrapped but IntelliJ defaults to a 120 character limit instead of 80.
I don't know of anyone who doesn't have screen estate for at least 120 characters these days so that default seems sensible to me.

Let me check how other OSS projects handle that.